Tao_Current.txt

```diff
@@ -400,29 +461,30 @@ One of the first things you must decide is whether you want your document to be
 
-    Publish the document as an Internet-Draft.
-    Receive comments on the draft.
-    Edit your draft based on the comments.
-    Repeat steps 1 through 3 a few times.
-    Ask an Area Director to take the draft to the IESG (if it's an individual submission). If the draft is an official Working Group product, the WG chair asks the AD to take it to the IESG.
-    If the Area Director accepts the submission, they will do their own initial review, and maybe ask for updates before they move it forwards.
-    Get reviews from the wider IETF membership. In particular, some of the Areas in the IETF have formed review teams to look over drafts that are ready to go to the IESG. Two of the more active review teams are from the Security Directorate ("SecDir") and the General Area Review Team (Gen-Art). Remember that all these reviews can help improve the quality of the eventual RFC.
-    Discuss concerns with the IESG members. Their concerns might be resolved with a simple answer, or they might require additions or changes to the document.
-    Wait for the document to be published by the RFC Editor.
-
-A much more complete explanation of these steps is contained in [BCP9], "The Internet Standards Process". Those who write drafts that they hope will become IETF standards must read BCP 9 so that they can follow the path of their document through the process. You can follow the progress on the IETF Datatracker http://datatracker.ietf.org. BCP 9 (and various other documents that update it) goes into great detail on a topic that is very often misunderstood, even by seasoned IETF participants: different types of RFCs go through different processes and have different rankings. There are six kinds of RFCs:
-
-    Proposed standards
-    Internet standards (sometimes called "full standards")
-    Best current practices (BCP) documents
-    Informational documents
-    Experimental documents
-    Historic documents
+    - Publish the document as an Internet-Draft.
+    - Receive comments on the draft.
+    - Edit your draft based on the comments.
+    - Repeat steps 1 through 3 a few times.
+    - Ask an Area Director to take the draft to the IESG (if it's an individual submission). If the draft is an official Working Group product, the WG chair asks the AD to take it to the IESG.
+    - If the Area Director accepts the submission, they will do their own initial review, and maybe ask for updates before they move it forwards.
+    - Get reviews from the wider IETF membership. In particular, some of the Areas in the IETF have formed review teams to look over drafts that are ready to go to the IESG. Two of the more active review teams are from the Security Directorate ("SecDir") and the General Area Review Team (Gen-Art), but there is also a Privacy Review Team and a Human Rights Review Team. Remember that all these reviews can help improve the quality of the eventual RFC.
```

Comment:
I think rather than try to list all of these, we should just remove the list :)

Reply:
I agree -- it would be quite long once opsdir and rtgdir are included

Reply:
Accepted and removed